podium-uaa/src/main/java/nl/thehyve/podium/service/UserService.java

```diff
@@ -167,6 +167,7 @@ private void copyProperties(UserRepresentation source, User target) {
         target.setDepartment(source.getDepartment());
         target.setJobTitle(source.getJobTitle());
         target.setSpecialism(source.getSpecialism());
+        target.setAdminVerified(source.isAdminVerified());
```

Comment:
The javadoc here says "Copy user properties, except login, password, email, activated.". This is because, e.g., the `copyProperties` function is also used for user registration. And you don't want a user to be able to set its account to be admin verified throught the registration form.
Admin verification should either be specifically added to the `updateUser` that is used by admins or a separate route.